site/src/job_queue/mod.rs

```diff
@@ -174,24 +177,84 @@ pub async fn build_queue(
     Ok(queue)
 }
 
+/// From a benchmark_request create all the required jobs
+pub fn create_benchmark_jobs(
```

Comment:
We can just pass the DB connection to this function and create the jobs immediately. If we go through the trouble of creating the actual job instance here, we might have as well sent it to the database function :laughing: I'd like to avoid having a constructor for `BenchmarkJob` to be publicly available.